README.md

```diff
@@ -56,19 +57,82 @@ schema](http://search.rcsb.org/rcsbsearch/v2/metadata/schema).
+          .and_("rcsb_struct_symmetry.kind", STRUCTURE_ATTRIBUTE_SEARCH_SERVICE).exact_match("Global Symmetry")
+          .and_("rcsb_entry_info.polymer_entity_count_DNA", STRUCTURE_ATTRIBUTE_SEARCH_SERVICE).greater_or_equal(1)
+          ).exec("assembly")
 
     # Exec produces an iterator of IDs
     for assemblyid in results:
         print(assemblyid)
 
+### Computed Structure Models
+
+The [RCSB PDB Search API](https://search.rcsb.org/#results_content_type)
+page provides information on how to include Computed Models into a search query. Here is a code example below.
+This query returns ID's for experimental and computed models associated with "hemoglobin". 
+Queries with only computed models or only experimental models can be made.
+    
+    from rcsbsearchapi.search import TextQuery
+    
+    q1 = TextQuery("hemoglobin")
+    
+    # add parameter as a list with either "computational" or "experimental" or both as list values
+    q2 = q1(return_content_type=["computational", "experimental"])
+    
+    list(q2)
+
+### Return Types and Attribute Search
+
+A search query can return different result types when a return type is specified. 
+Below are examples on specifying return types Polymer Entities,
+Non-polymer Entities, Polymer Instances, and Molecular Definitions, using a Structure Attribute query. 
+More information on return types can be found in the 
+[RCSB PDB Search API](https://search.rcsb.org/#building-search-request) page.
+
+    from rcsbsearchapi.search import AttributeQuery
+
+    q1 = AttributeQuery("rcsb_entry_container_identifiers.entry_id", "in", ["4HHB"]) # query for 4HHB deoxyhemoglobin
+
+    # Polymer entities
+    for poly in q1("polymer_entity"): # include return type as a string parameter for query object
+        print(poly)
+    
+    # Non-polymer entities
+    for nonPoly in q1("non_polymer_entity"):
+        print(nonPoly)
+    
+    # Polymer instances
+    for polyInst in q1("polymer_instance"):
+        print(polyInst)
+    
+    # Molecular definitions
+    for mol in q1("mol_definition"):
+        print(mol)
+
+### Protein Sequence Search Example
+
+Below is an example from the [RCSB PDB Search API](https://search.rcsb.org/#search-example-3) page,
+using the sequence search function.
+This query finds macromolecular PDB entities that share 90% sequence identity with
+GTPase HRas protein from *Gallus gallus* (*Chicken*).
+
+    from rcsbsearchapi.search import SequenceQuery
+
+    # Use SequenceQuery class and add parameters
+    results = SequenceQuery("MTEYKLVVVGAGGVGKSALTIQLIQNHFVDEYDPTIEDSYRKQVVIDGET" +
+                            "CLLDILDTAGQEEYSAMRDQYMRTGEGFLCVFAINNTKSFEDIHQYREQI" +
+                            "KRVKDSDDVPMVLVGNKCDLPARTVETRQAQDLARSYGIPYIETSAKTRQ" +
+                            "GVEDAFYTLVREIRQHKLRKLNPPDESGPGCMNCKCVIS", 1, 0.9)
+    
+    # Exec produces an iterator of IDs with return type - polymer entities
```

Comment:
I think you forgot to include the trailing `.exec()` :)

Reply:
`.exec()` is not necessary here since `results("polymer_entity")` handles that. But I can see how the comment might imply that.

Reply:
Ah OK, thanks for pointing out! In that case, then just updating the comment would be all